app/views/layouts/application.html.erb

```diff
@@ -1,9 +1,10 @@
 <!DOCTYPE html>
```

Comment:
Dodaj gem haml i przekonwertuj wszystkie widoki na haml'owe. Mozesz do tego uzyc np.:
https://github.com/dhl/erb2haml
https://github.com/haml/html2haml
Jak to zrobisz to zacznij pisac juz w hamlu.